src/uct/ib/rc/accel/rc_mlx5.inl

```diff
@@ -1190,9 +1191,8 @@ uct_rc_mlx5_iface_handle_expected(uct_rc_mlx5_iface_common_t *iface, struct mlx5
     uct_rc_mlx5_release_tag_entry(iface, tag_entry);
     uct_rc_mlx5_iface_tag_del_from_hash(iface, priv->buffer);
 
-    if (cqe->op_own & MLX5_INLINE_SCATTER_64) {
+    if (is_inline) {
```

Comment:
if it is not SW rendezvous we still need to do the copy

Reply:
I guess we don't need this with new changes as we are passing inline data to complete_cb

Reply:
correct